setup.cfg

```diff
@@ -6,16 +6,24 @@ markers =
 [yapf]
 based_on_style=google
 indent_width=2
-column_limit=80
-split_before_dict_set_generator=True
-coalesce_brackets=True
-# split_before_first_argument=True
+dedent_closing_brackets=True
+split_all_top_level_comma_separated_values=True
+split_before_first_argument=True
+split_penalty_after_opening_bracket=0
 
 [flake8]
 max-line-length=80
```

Comment:
I thought you were going to increase this? I'm cool with it like this, but just curious :)

Reply:
think we should stick to standard as much as possible, and since we'll be using a formatter I think it's less of an issue